docs/vagrant-spk/packaging-tutorial.md

```diff
@@ -196,9 +177,6 @@ the experience of using your app as other users.
 Over the next few steps, we will prepare the app so that it is visible in that
 Sandstorm instance.
 
-<!--(**Editor's note**: We should make localhost:6080 work, so that people don't have to learn about `local.sandstorm.io`.)-->
```

Comment:
Duplicate of the other comment, but same thing, this is still relevant, just the port is out of date.